lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :menu
+  attr_reader :basket
+
+  ERROR = "Item not found or unavailable"
+
+  def initialize(menu = Menu.new)
+    @basket = []
+    @menu = menu.display
+  end
+
+  def take(dish, quantity = 1)
```

Comment:
The `#take` method had me a little confused on what it does. `Order.take` doesn't` translate well for me.  Since it's being added to `@basket`, in my opinion `#add/add_to_ basket` would make it easier to understand what the method does at a glance.